app/src/main/java/io/actifit/fitnesstracker/actifitfitnesstracker/PostSteemitActivity.java

```diff
@@ -806,7 +806,7 @@ public void onCheckedChanged(RadioGroup group, int checkedId) {
                 getString(R.string.ChasingPokemons), getString(R.string.Badminton), getString(R.string.PickleBall),
                 getString(R.string.Snowshoeing),getString(R.string.Sailing),getString(R.string.Kayaking), getString(R.string.Kidplay),
                 getString(R.string.HomeImprovement), getString(R.string.YardWork), getString(R.string.StairClimbing),
-                getString(R.string.Yoga), getString(R.string.Stretching), getString(R.string.Plogging), getString(R.string.Crossfit),getString(R.string.FitnessGaming)
+                getString(R.string.Yoga), getString(R.string.Stretching),getString(R.string.Calisthenics),getString(R.string.StreetWorkout), getString(R.string.Plogging), getString(R.string.Crossfit),getString(R.string.Football)

```

Comment:
@AliHajjAli1 it seems you replaced fitnessgaming with football?

Reply:
Yes, maybe I copy it wrongly, I will fix it.

Reply:
still not properly typed @AliHajjAli1 , did the project compile for you? 
make sure to follow proper caps/small characters, should be typed as FitnessGaming as this is how its defined in the strings XML file

Reply:
Actually I created a new String so it compiled.
Now I fixed it